tests/foreman/api/test_docker.py

```diff
@@ -1278,77 +1287,74 @@ def test_positive_create_using_cv(self):
         self.assertEqual(len(content_view.version), 1)
         cvv = content_view.read().version[0].read()
         promote(cvv, lce.id)
-        for compute_resource in (self.cr_internal, self.cr_external):
-            with self.subTest(compute_resource):
-                container = entities.DockerHubContainer(
-                    command='top',
-                    compute_resource=compute_resource,
-                    organization=[self.org],
-                    repository_name=repo.container_repository_name,
-                    tag='latest',
-                    tty='yes',
-                ).create()
-                self.assertEqual(
-                    container.compute_resource.read().name,
-                    compute_resource.name
-                )
-                self.assertEqual(
-                    container.repository_name,
-                    repo.container_repository_name
-                )
-                self.assertEqual(container.tag, 'latest')
+
+        # publishing takes few seconds sometimes
+        sleep(10)
```

Comment:
ok, i've raised a BZ for the issue i was workarounding:
https://bugzilla.redhat.com/show_bug.cgi?id=1452149
and implemented the retry-if-bz-open approach